### PR TITLE
Assignment submission

### PR DESCRIPTION
**Pull request by jinghe315**

### CODE REVIEW

File: fix-code-0/main.cpp [hunk 1]
```diff
@@ -3,5 +3,6 @@
 using namespace std;
 
 int main() {
-  cout<<"Hello "; cout <<"World!"; cout << endl;
+  cout<<"Hello world!";
```
Comment (john-lakeman) [inline fix-code-0/main.cpp:6]: Recommend spaces before and after << for readability. 

Did this code run without a \n or endl?

File: fix-code-1/main.cpp [hunk 2]
```diff
@@ -3,5 +3,5 @@
 using namespace std;
 
 int main() {
-  cout << Hello;
+  cout << "Hello.";
```
Comment (john-lakeman) [inline fix-code-1/main.cpp:6]: \n"; or << endl; to end the line

File: fix-code-3/main.cpp [hunk 3]
```diff
@@ -3,5 +3,5 @@
 using namespace std;
 
 int main() {
-  cout >> "Hello";
+  cout << "Hello";
```
Comment (john-lakeman) [inline fix-code-3/main.cpp:6]: \n"; or << endl; to end the line

File: fix-code-4/main.cpp [hunk 4]
```diff
@@ -3,5 +3,5 @@
 using namespace std;
 
 int main() {
-  cout << "My GitHub username is [fill in your username here]!";
+  cout << "My GitHub username is jinghe315!";
```
Comment (john-lakeman) [inline fix-code-4/main.cpp:6]: \n"; or << endl; to end the line

File: invent-code-0/main.cpp [hunk 5]
```diff
@@ -3,5 +3,7 @@
 using namespace std;
 
 int main() {
-  // Write code here.
+std:: cout << "Hi, my name is Jing He.";
```
Comment (john-lakeman) [inline invent-code-0/main.cpp:6]: std:: not necessary

File: invent-code-0/main.cpp [hunk 6]
```diff
@@ -3,5 +3,7 @@
...
+std:: cout << "\nI live in Dallas, TX." ;
```
Comment (john-lakeman) [inline invent-code-0/main.cpp:7]: Interesting that you put the \n at the beginning.

File: invent-code-1/main.cpp [hunk 7]
```diff
@@ -3,5 +3,7 @@
 using namespace std;
 
 int main() {
-  // Write code here.
+std:: cout << "Hello again, my name is Jing.";
```
Comment (john-lakeman) [inline invent-code-1/main.cpp:6]: Nice to meet you!

File: invent-code-1/main.cpp [hunk 8]
```diff
@@ -3,5 +3,7 @@
...
+std:: cout << "\nI should be going to NYC soon.";
```
Comment (john-lakeman) [inline invent-code-1/main.cpp:7]: With the \n line breaks, I suspect they don't print on a single line.

std:: is unnecessary

File: fix-code-0/main.cpp (hunk 1 above)
Comment (jinghe315) [reply fix-code-0/main.cpp:6]: ok thanks! yes it ran without an \n or endl

File: fix-code-1/main.cpp (hunk 2 above)
Comment (jinghe315) [reply fix-code-1/main.cpp:6]: got it

File: fix-code-3/main.cpp (hunk 3 above)
Comment (jinghe315) [reply fix-code-3/main.cpp:6]: got it

File: fix-code-4/main.cpp (hunk 4 above)
Comment (jinghe315) [reply fix-code-4/main.cpp:6]: got it

File: invent-code-0/main.cpp (hunk 6 above)
Comment (jinghe315) [reply invent-code-0/main.cpp:7]: would you recommend putting it at the end?

File: invent-code-0/main.cpp (hunk 5 above)
Comment (jinghe315) [reply invent-code-0/main.cpp:6]: is std:: never necessary or is it just not necessary in this case?

File: invent-code-1/main.cpp (hunk 7 above)
Comment (jinghe315) [reply invent-code-1/main.cpp:6]: Nice to meet you too!

File: invent-code-1/main.cpp (hunk 8 above)
Comment (jinghe315) [reply invent-code-1/main.cpp:7]: got it

**Pull request closed without merging**